api/Element.json

```diff
@@ -8486,7 +8486,9 @@
           "spec_url": "https://wicg.github.io/sanitizer-api/#dom-element-sethtml",
           "support": {
             "chrome": {
-              "version_added": "105"
+              "version_added": "105",
+              "version_removed": "119",
+              "notes": "Existing implementation removed while specification update is worked on."
```

Comment:
I think this comment is redundant and can probably be removed.

Reply:
I think it is important, to make it clear that the technology is not just being dropped, just being temporarily removed. The "not deprecated" nature implies this, but I don't think it'll be that obvious to the average reader.

Reply:
It definitely looks a bit odd. I agree it isn't obvious to the average reader, I just don't see how that matters.

Reply:
I suppose to me it is the difference between the reader seeing it and going "OK, this isn't useful now, but it will be again in the future, I'll check back again later on", and the reader going "OK, Chromium browsers have removed this so this feature is dead in the water; I won't bother looking at this in the future" and possibly even "...oh wait, so shouldn't that be marked as deprecated then? What's going on?"

If you read this and _still_ don't think it matters, then I'll consider removing it, or just keeping it as page content. But yeah, my 2 cents.

Reply:
I'd rather it here than docs. BCD cleans itself up more regularly.

Reply:
I guess I read this note as an explanation for why the feature has been removed. Maybe it is fine to leave that explanation but if it is meant to make a promise about a come-back of this API, then it is a bit weird since I don't think we usually make such promises.

The Chromestatus says: "Since the final version of the standard will look different from our initial implementation, the goal is to prevent an API to become entrenched." 
So, I guess, we should absolutely discourage all readers from not using this API right now, but that messaging about the unstable spec should probably be on MDN?

Reply:
OK, you've talked some sense into me ;-)

Notes removed.